srcpkgs/exercism/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'exercism'
+pkgname=exercism
+version=3.0.10
+revision=1
+wrksrc="cli-${version}"
+build_style=go
+go_import_path="github.com/exercism/cli/exercism"
```

Comment:
this is wrong, the import path is the path to the github repository.

you meant 

```
go_import_path=github.com/exercism/cli
go_package="${go_import_path}/exercism"
```

Reply:
yeah, i misunderstood